nessie/jobs/create_metadata_schema.py

```diff
@@ -44,11 +44,4 @@ def run(self):
             raise BackgroundJobError(f'RDS metadata schema creation failed.')
             return False
 
-        resolved_ddl_redshift = resolve_sql_template('create_metadata_schema_redshift.template.sql')
```

Comment:
@raydavis - Is `create_metadata_schema_redshift.template.sql` still used?

Reply:
Not by code, @johncrossman , so I should delete it, even if the Redshift config variables stay around for a release. I'll amend.